roles/snpseq-metadata-service/tasks/install_package.yml

```diff
@@ -0,0 +1,19 @@
+
```

Comment:
Is there a reason why the snpseq_metadata package is not installed as a dependency of the snpseq_metadata service?

And also, shouldn't you download the ENA/SRA xml schema after installing the package here?

Reply:
not other than I opted for not creating a separate role for `snpseq_metadata` since it will likely only be used by this service and also installed in the same environment, so I figured I could just as well deploy them together.

good catch about the schema, I will definitely need that, yes 🤦

Reply:
...or I guess you maybe mean that it should be a python dependency? that could perhaps be an option, not sure how straightforward it would be to run the post-install stuff but I guess there's probably hooks for that in pyproject